code/modules/organs/internal/brain.dm

```diff
@@ -15,50 +15,55 @@
 	relative_size = 85
 	damage_reduction = 0
 	scale_max_damage_to_species_health = FALSE
-
-	var/can_use_mmi = TRUE
-	var/mob/living/carbon/brain/brainmob = null
-	var/const/damage_threshold_count = 10
-	var/damage_threshold_value
-	var/healed_threshold = 1
+	transfer_brainmob_with_organ = TRUE
+	var/can_use_brain_interface = TRUE
+	var/should_announce_brain_damage = TRUE
 	var/oxygen_reserve = 6
 
+	var/VAR_PRIVATE/mob/living/brain/_brainmob
+
+/obj/item/organ/internal/brain/get_brainmob(var/create_if_missing = FALSE)
+	if(!_brainmob && create_if_missing)
+		_brainmob = new(src)
+	return _brainmob
+
+/obj/item/organ/internal/brain/Initialize()
+	. = ..()
+	if(species)
+		set_max_damage(species.total_health)
+	else
+		set_max_damage(200)
+	initialize_brain()
```

Comment:
Should this be here? Before, brainmobs were only made in `transfer_identity`; it looks like this is why tests are failing, and besides, `create_if_missing` is used in the new equivalent of the `transfer_identity` proc. Why is it necessary to forcibly create a brainmob here, which has to register to its parent + creates a virtual mob + may cause all other sorts of mayhem?